src/sentry/api/serializers/models/group.py

```diff
@@ -526,7 +526,8 @@ class GroupStatsMixin(object):
         "24h": StatsPeriod(24, timedelta(hours=1)),
     }
 
-    CUSTOM_PERIOD_SEGMENTS = 29  # for 30 segments use 1/29th intervals
+    CUSTOM_SEGMENTS = 29  # for 30 segments use 1/29th intervals
+    CUSTOM_ROLLUP_6H = timedelta(hours=6).total_seconds()  # rollups should be increments of 6hs
```

Comment:
Might be better to just hard code this as 21600

Reply:
Isn't this more readable?

Reply:
I like the timedelta thing personally. You'll want to cast this as an int though, since `total_seconds()` is a float.

Reply:
I kind of think making this `SIX_HOURS=21600` and using that variable would feel more readable. Maybe a `TWENTYFOUR_HOURS=SIX_HOURS*4` and using that to replace `timedelta(hours=24).total_seconds()` as well.

Reply:
Your call though, I'm happy with this going through either way.

Reply:
I cast to int at the end after divisions